pkg/http/core/kubernetes/scale_test.go

```diff
@@ -84,6 +84,17 @@ var _ = Describe("Scale", func() {
 		})
 	})
 
+	When("the kind is not supported to scale", func() {
+		BeforeEach(func() {
+			actionConfig.Operation.ScaleManifest.ManifestName = "not-supported-kind test-name"
```

Comment:
Shuldnt this be?:
actionConfig.Operation.ScaleManifest.ManifestName = “not-supported-kind”?

to match the message in Line #94